modules/tinymce/src/plugins/table/main/ts/ui/CellDialog.ts

```diff
@@ -42,7 +42,7 @@ const updateSimpleProps = (modifier: DomModifier, colModifier: DomModifier, data
   if (shouldUpdate('scope')) {
     modifier.setAttrib('scope', data.scope);
   }
-  if (shouldUpdate('class')) {
+  if (shouldUpdate('class') && data.class !== 'mce-no-match') {
```

Comment:
I guess this new value needs to be documented right if you configure a custom class list you might want to have this one as one of the options where it doesn't change the class value.